contracts/kleros/TalentLayerMultipleArbitrableTransaction.sol

```diff
@@ -129,39 +129,48 @@ contract MultipleArbitrableTransaction is IArbitrable {
         jobRegistryAddress = _jobRegistryAddress;
     }
 
+    function getAmount(uint256 _jobId, uint256 _proposalId) private view returns (uint256){
```

Comment:
so to prevent multiple externals calls, we can request one time the whole proposal and job, and then access to properties we need